src/hotspot/share/oops/constantPool.cpp

```diff
@@ -942,7 +942,7 @@ BasicType ConstantPool::basic_type_for_constant_at(int which) {
 // Some constant pool entries cache their resolved oop. This is also
 // called to create oops from constants to use in arguments for invokedynamic
 oop ConstantPool::resolve_constant_at_impl(const constantPoolHandle& this_cp,
-                                           int index, int cache_index,
+                                           int cp_index, int cache_index,
```

Comment:
Not sure that we really need to always say `cp_index` given the context. Changing `which` is good, but `index` was perfectly fine IMO.